crates/consensus/src/transaction/eip1559.rs

```diff
@@ -288,6 +276,18 @@ impl Transaction for TxEip1559 {
     fn gas_price(&self) -> Option<u128> {
         None
     }
+
+    fn to(&self) -> TxKind {
+        self.to
+    }
+
+    fn value(&self) -> U256 {
+        self.value
+    }
+
+    fn input(&self) -> &[u8] {
+        &self.input
+    }
```

Comment:
Goal of these changes is cosmetic, simply enforcing consistent ordering for clarity